oauth2client/contrib/django_util/models.py

```diff
@@ -16,6 +16,12 @@
 
 import base64
 import pickle
+import sys
+
+try:
+    import jsonpickle
+except ImportError:
+    pass
```

Comment:
I think I already made jsonpickle a dependency of this contrib package? Have you tested it with regular pickle? Think there are some subtle issues...maybe best to just require jsonpickle.